.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+
+jobs:
+  tests:
+    docker:
+      - image: circleci/python:3.7-browsers
+      - image: circleci/mysql:5.7
```

Comment:
I retrieved the mysql version from an existing build configuration, is it correct? should it be upgraded ?

Reply:
5.7 is the minimum supported version (the one in Ubuntu 18.04, so a stable & not bleeding edge version) ; it's good to me.